lib/carin_for_blue_button_test_kit/client/collection.rb

```diff
@@ -0,0 +1,19 @@
+module CarinForBlueButtonTestKit
+  SET_TO_BE_GATHERED = { Coverage: ['c4bb-Coverage'],
+                         ExplanationOfBenefit: ['c4bb-EOBInpatient', 'c4bb-EOBOral', 'c4bb-EOBOutpatient',
+                                                'c4bb-EOBPharmacy', 'c4bb-EOBProfessional'],
+                         Organization: ['c4bb-Organization'],
+                         Patient: ['998'],
+                         Practitioner: ['c4bb-Practitioner'],
+                         RelatedPerson: ['c4bb-RelatedPerson'] }.freeze
+  # TODO: What about Provenance?
+
+  SEARCHES_BY_PRIORITY = { Coverage: [['_id'], ['_lastUpdated']],
```

Comment:
I think we need to have patient as an allowed (eventually required) search parameter here. It is what US Core requires now even though there isn't a Coverage resource in the 3.1.1 version that Carin references and it just doesn't make any sense without allowing a search by patient. Probably can handle as a part of [FI-3356](https://oncprojectracking.healthit.gov/support/browse/FI-3356).